src/tic_tac_toe/play_game.clj

```diff
@@ -1,5 +1,7 @@
 (ns tic-tac-toe.play-game) 
 (use '[tic-tac-toe.console])
+(use '[tic-tac-toe.board])
 
 (defn -main []
-  (print-empty-board))
+ (print-welcome-message) (print-empty-board) (ask-for-move) (print-board (make-move ["-" "-" "-" "-" "-" "-" "-" "-" "-"] (get-move) "X")))
```

Comment:
as mentioned in the previous PR, try and give namespaces an alias so it's obvious where functions are being called from. this blog from Colin talks about how to use aliases in clojure

https://blog.8thlight.com/colin-jones/2010/12/05/clojure-libs-and-namespaces-require-use-import-and-ns.html